src/server/debug2.rs

```diff
@@ -412,27 +381,81 @@ impl<ER: RaftEngine> Debugger for DebuggerImplV2<ER> {
     }
 
     fn dump_kv_stats(&self) -> Result<String> {
-        unimplemented!()
+        let mut kv_str = String::new();
+        self.tablet_reg.for_each_opened_tablet(|_, cached| {
+            if let Some(tablet) = cached.latest() {
+                let str = MiscExt::dump_stats(tablet).unwrap();
+                kv_str.push_str(&str);
+            }
+            true
+        });
+        if let Some(s) = self.kv_statistics.as_ref() && let Some(s) = s.to_string() {
+            kv_str.push_str(&s);
+        }
+        Ok(kv_str)
     }
 
     fn dump_raft_stats(&self) -> Result<String> {
-        unimplemented!()
+        let mut raft_str = box_try!(RaftEngine::dump_stats(&self.raft_engine));
+        if let Some(s) = self.raft_statistics.as_ref() && let Some(s) = s.to_string() {
+            raft_str.push_str(&s);
+        }
+        Ok(raft_str)
     }
 
-    fn modify_tikv_config(&self, _config_name: &str, _config_value: &str) -> Result<()> {
-        unimplemented!()
+    fn modify_tikv_config(&self, config_name: &str, config_value: &str) -> Result<()> {
+        if let Err(e) = self.cfg_controller.update_config(config_name, config_value) {
+            return Err(Error::Other(
+                format!("failed to update config, err: {:?}", e).into(),
+            ));
+        }
+        Ok(())
     }
 
     fn get_store_ident(&self) -> Result<StoreIdent> {
-        unimplemented!()
+        self.raft_engine
+            .get_store_ident()
+            .transpose()
+            .unwrap()
+            .map_err(|e| Error::EngineTrait(e))
     }
 
-    fn get_region_properties(&self, _region_id: u64) -> Result<Vec<(String, String)>> {
-        unimplemented!()
-    }
+    fn get_region_properties(&self, region_id: u64) -> Result<Vec<(String, String)>> {
+        let region_state = self
+            .raft_engine
+            .get_region_state(region_id, u64::MAX)
+            .unwrap()
+            .unwrap();
```

Comment:
Nit: suggest to return `Error::NotFound` on `None` for more friendly.

Reply:
fixed